Test/Microsoft.ApplicationInsights.Test/Shared/Extensibility/Implementation/Experimental/ExperimentalFeaturesTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Experimental
+{
+    [TestClass]
+    public class ExperimentalFeaturesTests
+    {
+        [TestMethod]
+        public void VerifyExperimentalFeaturesExtensionIfConfigEmpty()
+        {
+            var telemetryConfiguration = new TelemetryConfiguration();
+
+            bool value = telemetryConfiguration.EvaluateExperimentalFeature("abc");
+            Assert.IsFalse(value);
+        }
+
+        [TestMethod]
+        public void VerifyExperimentalFeaturesExtension()
+        {
+            var telemetryConfiguration = new TelemetryConfiguration
+            {
+                ExperimentalFeatures = { "abc" }
+            };
+
+            bool value = telemetryConfiguration.EvaluateExperimentalFeature("abc");
+            Assert.IsTrue(value);
+
+            bool fakeValue = telemetryConfiguration.EvaluateExperimentalFeature("fake");
+            Assert.IsFalse(fakeValue);
+        }
+
+        [TestMethod]
+        public void VerifyCanAddExperimentalFeatures()
+        {
+            var telemetryConfiguration = new TelemetryConfiguration
+            {
+                ExperimentalFeatures = { "abc" }
```

Comment:
this probably works because of InternalsVisibleTo for test projects. Can we avoid calling internal APIs?

Reply:
If we're not going to use it in Test classes, I would recommend removing the `internal set`.
Only the BaseSDK repo will be able to use internals. 
We don't make internals available to the other repos.

What do you think?

Reply:
I agree that we should remove `internal set` if the only purpose was test code